zipline/algorithm.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, *args, **kwargs):
         self.algoscript = kwargs.pop('script', None)
 
         self._initialize = None
+        self._analyze = kwargs.pop('analyze', None)
```

Comment:
> I should have checked this more closely. Perhaps a test is warranted.

That's a great idea - want to add a test to, say, `test_algorithm.py`?

> I had it working from the elif previously, but moved per a previous suggestion. Perhaps I misinterpreted the exact location. 

Sorry about that.  I was thinking we should put it at the end of the block that currently looks like:
```python
             self._initialize = kwargs.pop('initialize')
             self._handle_data = kwargs.pop('handle_data')
             self._before_trading_start = kwargs.pop('before_trading_start',
                                                     None)
```
> By the way is there a specific branch I should use for the PR?

When upstreaming changing, my workflow is to create (in my fork) a new branch with a descriptive name, PR it against the upstream master, like you've done here, and then once it's merged, I update my fork's master with the changes from upstream/master.  That way my master is always in sync with upstream.  (I delete the branch afterwards.)